tests/test_helpers.py

```diff
@@ -145,10 +145,65 @@ def test_deserialize_valid_5(self) -> None:
         )
 
     def test_deserialize_valid_6(self) -> None:
-        self.assertEqual(
-            XsdDateTime.deserialize('2001-10-26T21:32:52.12679'),
-            datetime(year=2001, month=10, day=26, hour=21, minute=32, second=52, microsecond=12679, tzinfo=None)
-        )
+        """Test that less than 6 decimal places in the seconds field is padded to 6 and parsed correctly."""
+
+        # Use mock wrapping datetime so we can check that it was given a correctly padded string
+        with mock.patch('serializable.helpers.datetime', wraps=datetime) as datetime_mock:
+
+            # Make sure the parsed datetime is what we expect
+            self.assertEqual(
+                XsdDateTime.deserialize('2001-10-26T21:32:52.12679'),
+                datetime(year=2001, month=10, day=26, hour=21, minute=32, second=52, microsecond=12679, tzinfo=None)
```

Comment:
actually, it is expected to be `126790` **microsecond**.
microsecond is not a fraction, but a range between 0 and 999999.

this means, 1.337 has 337000 microns

Reply:
fixed via 870a98a6da913232fdfaada83fe4a8130e2a888e